docs/governance/community-guidelines/releases/cmb.md

```diff
@@ -0,0 +1,106 @@
+# Change Management Board
+
+This document is a [community guideline].
+
+## Purpose
+
+The document outlines and defines the guidelines for the Change Management Board (CMB) for the Common Cloud Controls (CCC) project.
+
+The CMB is a body of representatives from financial institutions of varying sizes and types. Its primary role is to review and approve changes and new catalogs that are within the Release Candidate. The CMB collectively represents end-user stakeholders, ensuring that each artifact is adaptable to the needs of a wide range of institutions while maintaining consistency and integrity across the board.
+
+## Membership
+
+The change management board is comprised of a Release Manager and the body of reviewers.
```

Comment:
"comprises" or "is composed of" but not "is comprised of"! I favour "is composed of" in this situation.

Reply:
```suggestion
The change management board is composed of a Release Manager and the body of reviewers.
```